### PR TITLE
add switch on put API

### PR DESCRIPTION
**Pull request by tigershi**

### CODE REVIEW

File: g11n-ws/vip-manager-lite-i18n/src/main/resources/application-bundle.properties [hunk 1]
```diff
@@ -28,6 +28,9 @@ server.trace.enable=false
 server.shutdown=graceful
 spring.lifecycle.timeout-per-shutdown-phase=30s
 
+#translation update switch
+translation.update.enable=true
```
Comment (Xiaochao8) [inline g11n-ws/vip-manager-lite-i18n/src/main/resources/application-bundle.properties:32]: According to our discussion, this is false by default?

Comment (tigershi) [reply g11n-ws/vip-manager-lite-i18n/src/main/resources/application-bundle.properties]: change it to false

File: g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v1/translation/TranslationSyncAPI.java [hunk 2]
```diff
@@ -29,6 +30,7 @@
  *
  */
 @RestController
+@ConditionalOnProperty(name = "translation.update.enable", havingValue = "true", matchIfMissing = true)
```
Comment (huihuiw01) [inline g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v1/translation/TranslationSyncAPI.java:33]: Should matchIfMissing = false?

File: g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/translation/TranslationSyncAPI.java [hunk 3]
```diff
@@ -30,6 +31,7 @@
  *
  */
 @RestController("v2-TranslationSyncAPI")
+@ConditionalOnProperty(name = "translation.update.enable", havingValue = "true", matchIfMissing = true)
```
Comment (huihuiw01) [inline g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/translation/TranslationSyncAPI.java:34]: Should matchIfMissing = false?

**Review by jin-jasper — APPROVED** [review]

**Review by huihuiw01 — APPROVED** [review]

File: g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/translation/TranslationSyncAPI.java (hunk 3 above)
Comment (tigershi) [reply g11n-ws/modules/md-restful-api-i18n/src/main/java/com/vmware/vip/i18n/api/v2/translation/TranslationSyncAPI.java:34]: yes

**Pull request merged**